spec/tictactoe/validation_spec.rb

```diff
@@ -1,47 +1,64 @@
 require "./lib/tictactoe/validation"
+require "./lib/tictactoe/messages"
+require "./spec/tictactoe/fakers"
+require "./lib/tictactoe/output_input"
 
 RSpec.describe Tictactoe::Validation do
   context "check_position_range" do
     it "should not allow players to play using positions out of range" do
-      validate = Tictactoe::Validation.new
+      messages = Tictactoe::Messages.new(Tictactoe::ALL_MESSAGES, 0)
+      io = Tictactoe::OutputInput.new()
+      validate = Tictactoe::Validation.new(messages,io)
```

Comment:
A lot of the test setup code seems duplicated, check out this [before](https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/before-and-after-hooks) method from RSpec.